### ISSUE TITLE
For Kernel 5.3 "nixos-rebuild switch" failing  with no support for zfsonlinux

### ISSUE THREAD

**Issue by mudrii** [NixOS/nixpkgs#69187]
**Describe the bug**
After system NixOs was updated to latest Linux Kernel 5.3 can run anymore update on the system

```sh
nixos-rebuild switch --upgrade
unpacking channels...
building Nix...
building the system configuration...
error: Linux v5.3 is not yet supported by zfsonlinux v0.8.1.
Try zfsUnstable or set the NixOS option boot.zfs.enableUnstable.

(use '--show-trace' to show detailed location information)
```
Note: I am not using zfs or have any configuration referring to zfs
Note 2: System is on the unstable channel

**To Reproduce**
1. in configuration.nix
boot.kernelPackages = pkgs.linuxPackages_latest;
2. After kernel update run 
nixos-rebuild switch --upgrade

**Expected behavior**
Update system with latest packages or update with changes done in a configuration files

**Additional context**

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
```sh
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.0, NixOS, 20.03pre193701.262b328b0ba (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-20.03pre193781.d484f2b7fc0"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by vcunat** [comment]
> Note: I am not using zfs or have any configuration referring to zfs

IIRC we now ship ZFS support by default, so I believe that's why.

**Comment by vcunat** [comment]
Fixed by 53e1d838eb29f4988b6d13b1626ea13f39d2150e.
